File: questions/command/haskell/index/src/Command.hs
module Command where

type T = String

perfect :: T
perfect = "perfect"

great :: T
great = "great"

good :: T
good = "good"

bad :: T
bad = "bad"

miss :: T
miss = "miss"

isEqualPerfect :: T -> Bool
isEqualPerfect s = s == perfect

isEqualGreat :: T -> Bool
isEqualGreat s = s == great

File: questions/command/clojure/index/src/index/command.clj
(ns index.command)

(def perfect "perfect")
(def great "great")
(def good "good")
(def bad "bad")
(def miss "miss")

(defn is-equal-perfect [s]
  (= s perfect))

(defn is-equal-great [s]
  (= s great))
